### PR TITLE
Update troubleshooting.md log file & msvcr

### PR DESCRIPTION
**Pull request by lizardwithawizard**
Finally got around to adding `msvcr120` to troubleshooting, originally mentioned in #107
Updated log file section to add instructions locally to docs instead of sending users to third party site (users did not go to the site and always got confused on where to look)
Removed server full section as it shouldn't be relevant anymore

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Actually had the `MSVCR120.dll`/`MSVCP120.dll` error myself the other day due to a clean install. Luckily I even took notes and screenshots while resolving.

I'll revisit this one later today when I'm back on my main machine to double check PR with my notes and add screenshots ^^

File: docs/installing-northstar/troubleshooting.md [hunk 1]
```diff
@@ -107,15 +107,13 @@ See also [this thread on Steam](https://steamcommunity.com/app/1237970/discussio
...
 
-If you get this error you can fix it by [installing vcredist 2015-2022](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170). If it does not work install the 2013 version
+If you get this error you can fix it by [installing vcredist 2013](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2013-vc-120).
 
-## Server ALWAYS Full <a href="#servers-full" id="servers-full"></a>
+## VCRUNTIME140 Not found <a href="#vcruntime" id="vcruntime"></a>
 
-* Double click on the server name in rapid succession.
-* Alternatively, click on the server you want to join, and click "join".
-  * Clicking on no server but clicking "join" will attempt to connect you to the very first server, which is usually full
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: Kinda think removing this section should be separate PR cause it messes with diff

File: docs/installing-northstar/troubleshooting.md [hunk 2]
```diff
@@ -201,6 +199,6 @@ Open console in-game in main menu, type in `ns_resetpersistence` and press enter
 
 The default install location for EA App `C:\Program Files\EA Games\Titanfall2` is not writeable by non-admin processes. This messes with Northstar trying to write log files as well as mod-managers trying to install mods.
 
-Therefore the recommended solution is to move the install to another location (can even be on the same drive). This prevents the non-admin issue and as such should solve the issue of Northstar being unable to write logs and failing.
+Therefore the recommended solution is to move the install to another location (can even be on the same drive). This prevents the non-admin issue and as such should solve the issue of Northstar being unable to write logs and failing.
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
Therefore the recommended solution is to move the install to another location (can even be on the same drive). This prevents the non-admin issue and as such should solve the issue of Northstar being unable to write logs and failing.
```

Trailing space added accidentally here

File: docs/installing-northstar/troubleshooting.md [hunk 3]
```diff
@@ -201,6 +199,6 @@ Open console in-game in main menu, type in `ns_resetpersistence` and press enter
...
+Therefore the recommended solution is to move the install to another location (can even be on the same drive). This prevents the non-admin issue and as such should solve the issue of Northstar being unable to write logs and failing. 
 
-See also [https://www.pcmag.com/how-to/how-to-move-pc-games-to-a-different-hard-drive](https://www.pcmag.com/how-to/how-to-move-pc-games-to-a-different-hard-drive) (scroll down to "_EA: Move the Folder and Reinstall_")
+The recommended solution can be done by moving the Titanfall 2 folder from the default location to something like `C:\Program Files (x86)\Games\Titanfall2`, and then you will need to open EA App, go to Settings, go to Downloads and change the install directory in the settings. Click Edit next to Install Location and navigate to your new directory that you put Titanfall 2 into. Restart EA App after doing this, and the issue should be resolved. If not, hitting Install on Titanfall 2 should prompt EA App to look for the files, or ask you to set the game's directory.
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: I agree that not (being able to) link to the right section on the 3rd party site is annoying but we should still have a visual reference. Could copy-paste the screenshot in the article for example ;)

Also this section should have a line break somewhere, makes it easier to read.

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
A few first comments

File: docs/installing-northstar/troubleshooting.md [hunk 4]
```diff
@@ -107,15 +107,13 @@ See also [this thread on Steam](https://steamcommunity.com/app/1237970/discussio
 
 Go to your `Titanfall2` directory. From there go to the `R2Northstar` and delete `enabledmods.json`. This file stores information about which mods are enabled and disabled. By deleting this file Northstar will fall back to the default (all mods enabled) and re-create the file appropriately.
 
-## VCRUNTIME140 Not found <a href="#vcruntime" id="vcruntime"></a>
+## MSVCR120.dll / MSVCP120.dll Not found <a href="#msvcr" id="msvcr"></a>
 
-If you get this error you can fix it by [installing vcredist 2015-2022](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170). If it does not work install the 2013 version
+If you get this error you can fix it by [installing vcredist 2013](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2013-vc-120).
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: When I did it for myself I found that the `vcredist_x64.exe` is enough. So maybe we should mention just downloading that one ^^

**Comment by GeckoEidechse** [comment]
Found the screenshots for the error messages I had. Probably best to add them to the corresponding section as well ^^

msvcp120dll
msvcr120dll

File: docs/installing-northstar/troubleshooting.md (hunk 1 above)
Comment (lizardwizardonthewall) [reply docs/installing-northstar/troubleshooting.md]: #118 made to resolve this

File: docs/installing-northstar/troubleshooting.md (hunk 4 above)
Comment (lizardwizardonthewall) [reply docs/installing-northstar/troubleshooting.md]: mentioned specifically x64 in the section

File: docs/installing-northstar/troubleshooting.md (hunk 3 above)
Comment (lizardwizardonthewall) [reply docs/installing-northstar/troubleshooting.md]: added image from website for a visual reference, broke section up for easier reading

File: docs/installing-northstar/troubleshooting.md [hunk 5]
```diff
@@ -89,10 +89,17 @@ See also [this thread on Steam](https://steamcommunity.com/app/1237970/discussio
 
 Go to your `Titanfall2` directory. From there go to the `R2Northstar` and delete `enabledmods.json`. This file stores information about which mods are enabled and disabled. By deleting this file Northstar will fall back to the default (all mods enabled) and re-create the file appropriately.
 
+## MSVCR120.dll / MSVCP120.dll Not found <a href="#msvcr" id="msvcr"></a>
+
+If you get this error you can fix it by [installing vcredist 2013 (vcredist_x64)](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2013-vc-120).
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
If you get this error you can fix it by [downloading and installing vcredist 2013 (`vcredist_x64.exe`)](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2013-vc-120).
```

File: docs/installing-northstar/troubleshooting.md [hunk 6]
```diff
@@ -89,10 +89,17 @@ See also [this thread on Steam](https://steamcommunity.com/app/1237970/discussio
...
+
+![MSVCR120.dll Error](../images/msvcr.png)
+
 ## VCRUNTIME140 Not found <a href="#vcruntime" id="vcruntime"></a>
 
-If you get this error you can fix it by [installing vcredist 2015-2022](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170). If it does not work install the 2013 version
+If you get this error you can fix it by [installing vcredist 2015-2022](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022). If it does not work install the 2013 version
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: Do you know if it's the `x64` version again?

**EDIT:** I think I had a stroke while writing the comment the first time

File: docs/installing-northstar/troubleshooting.md [hunk 7]
```diff
@@ -89,10 +89,17 @@ See also [this thread on Steam](https://steamcommunity.com/app/1237970/discussio
...
 
+main
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion

```

Leftover from dealing with merge conflict ig? ^^

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]

File: docs/installing-northstar/troubleshooting.md [hunk 8]
```diff
@@ -185,4 +192,7 @@ The default install location for EA App `C:\Program Files\EA Games\Titanfall2` i
 
 Therefore the recommended solution is to move the install to another location (can even be on the same drive). This prevents the non-admin issue and as such should solve the issue of Northstar being unable to write logs and failing.
 
-See also [https://www.pcmag.com/how-to/how-to-move-pc-games-to-a-different-hard-drive](https://www.pcmag.com/how-to/how-to-move-pc-games-to-a-different-hard-drive) (scroll down to "_EA: Move the Folder and Reinstall_")
+The recommended solution can be done by moving the Titanfall 2 folder from the default location to something like `C:\Program Files (x86)\Games\Titanfall2`. After doing this, you will need to open the EA App, go to Settings, go to Downloads and change the install directory in the settings. Click Edit next to Install Location and navigate to your new directory that you put Titanfall 2 into.
+![EA App Settings](../images/ea-app.png)
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
The recommended solution can be done by moving the Titanfall 2 folder from the default location to something like `C:\Program Files (x86)\Games\Titanfall2`. After doing this, you will need to open the EA App, go to Settings, go to Downloads and change the install directory in the settings. Click _"Edit"_ next to _"Install Location"_ and navigate to your new directory that you put Titanfall 2 into.

![EA App Settings](../images/ea-app.png)
```

Formatting and I think a newline is needed here ^^

File: docs/installing-northstar/troubleshooting.md [hunk 9]
```diff
@@ -185,4 +192,7 @@ The default install location for EA App `C:\Program Files\EA Games\Titanfall2` i
...
+
+Restart EA App after doing this, and the issue should be resolved. If not, hitting Install on Titanfall 2 should prompt EA App to look for the files, or ask you to set the game's directory.
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
Restart EA App after doing this, and the issue should be resolved. If not, hitting _Install_ on Titanfall 2 should prompt EA App to look for the files, or ask you to set the game's directory.
```

Formatting ^^

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]

File: docs/installing-northstar/troubleshooting.md (hunk 7 above)
Comment (lizardwizardonthewall) [reply docs/installing-northstar/troubleshooting.md]: Probably. (most likely)

File: docs/installing-northstar/troubleshooting.md (hunk 6 above)
Comment (lizardwizardonthewall) [reply docs/installing-northstar/troubleshooting.md]: ...yes?

Comment (lizardwizardonthewall) [reply docs/installing-northstar/troubleshooting.md]: I think I added the mention in mscvr120 and forgot to add it to this one, whoops :sweat_smile:

Comment (lizardwizardonthewall) [reply docs/installing-northstar/troubleshooting.md]: done & done

File: docs/installing-northstar/troubleshooting.md [hunk 10]
```diff
@@ -89,10 +89,15 @@ See also [this thread on Steam](https://steamcommunity.com/app/1237970/discussio
...
+
+If you get this error you can fix it by [downloading and installing vcredist 2013 (`vcredist_x64.exe`)](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2013-vc-120).
+
+![MSVCR120.dll Error](../images/msvcr.png)
 
-If you get this error you can fix it by [installing vcredist 2015-2022](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170). If it does not work install the 2013 version
+## VCRUNTIME140 Not found <a href="#vcruntime" id="vcruntime"></a>
 
+If you get this error you can fix it by [installing vcredist 2015-2022 (`vcredist_x64.exe`)](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022). If it does not work install the 2013 version
 ## Launch Northstar Locked <a href="#locked-northstar" id="locked-northstar"></a>
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md:102]:
```suggestion

## Launch Northstar Locked <a href="#locked-northstar" id="locked-northstar"></a>
```

Just noticed a missing newline here. Might have introduced by a suggested change of mine, whoops. I'll fix it ^^

**Review by GeckoEidechse — APPROVED** [review]
Aight looks good now ^^
